src/Integration.UnitTests/Helpers/OutputWindowServiceTests.cs

```diff
@@ -68,61 +71,19 @@ public void Show_FailedToActivateSonarLintPane_OutputWindowNotShown()
             var serviceProvider = new Mock<IServiceProvider>();
             SetupSonarLintPane(serviceProvider, sonarLintPane.Object);
 
-            var testSubject = new OutputWindowService(serviceProvider.Object);
+            var toolWindowServiceMock = new Mock<IToolWindowService>();
 
-            using (new AssertIgnoreScope())
-            {
-                testSubject.Show();
-            }
-
-            serviceProvider.VerifyAll();
-            serviceProvider.VerifyNoOtherCalls();
-        }
-
-        [TestMethod]
-        public void Show_ActivatedSonarLintPane_FailedToGetOutputWindow_OutputWindowNotShown()
```

Comment:
The tests are simpler now; they don't need to set up IVsUIShell since that is now hidden inside the tool window service.
The only thing the tests need to check now is whether the tool window service is called or not.